src/site/content/en/fast/use-lighthouse-for-performance-budgets/index.md

```diff
@@ -0,0 +1,65 @@
+# Use Lighthouse for Performance Budgets
+
+Lighthouse now supports performance budgets. This feature, LightWallet, can be set up in under five minutes and provides feedback on the size and quantity of page resources.
+
+## Install Lighthouse
+
+LightWallet is available in the comand line version Lighthouse (versions 5 & up). 
+
+Install Lighthouse:
+
+```bash
+$ npm install -g lighthouse
+```
+
+## Create a Budget
+
+Create a file named `budget.json`. In this file add the following JSON:
+
+```json
+[
+    {
+        "resourceSizes": [
+            {
+                "resourceType": "script",
+                "budget": 125
+            },
+            {
+                "resourceType": "total",
+                "budget": 300
+            }
+        ],
+        "resourceCounts": [
+            {
+                "resourceType": "third-party",
+                "budget": 10
+            }
+        ]
+    }
+]
+```
+
+This example budget.json file sets three separate budgets:
+
+*   A budget of 125 KB for the total amount of JavaScript on the page.
+*   A budget of 300 KB for the overall size of the page.
+*   A budget of 10 requests for the numer of requests made to third-party origins.
+
+A `budget.json` file can contain any number of budgets. These are the resource types supported by LightWallet: `document`, `font`, `image`, `media`, `other`,`script`, `stylesheet`, `third-party`, and `total`.
+
+
+## Run Lighthouse
+
+Run Lighthouse using the `--budget-path` flag. This flag tells Lighthouse the location of your budget file. (Note: A budget file does not have to be named `budget.json`.)  
```

Comment:
Suggestion to create an aside of type note and put 'A budget file does not have to be named...', rather than having it built into text.

I'd also suggest putting it after the sample bash.